app/_src/gateway/reference/configuration.md

```diff
@@ -2375,8 +2375,9 @@ Turn on FIPS mode; this mode is only available on a FIPS build.
 
 The Admin GUI for Kong Enterprise.
```

Comment:
Please remove the badge changes from this file (`configuration.md`). The badges have been added in https://github.com/Kong/docs.konghq.com/pull/5881, along with the script update that'll make sure the right badges get picked up in subsequent versions.